llvm/lib/AsmParser/LLLexer.cpp

```diff
@@ -262,6 +267,28 @@ void LLLexer::SkipLineComment() {
   }
 }
 
+/// SkipCComment - This skips C-style /**/ comments. Returns true if there
```

Comment:
```suggestion
/// This skips C-style /**/ comments. Returns true if there
```
Don't repeat function name in doc comments. (This is an old style that shouldn't be replicated.)